CHANGELOG.md

```diff
@@ -8,6 +8,9 @@
 **New Feature:**
 - Add method to dynamically set namespace in yaml utility [kubernetes-client/python#782](https://github.com/kubernetes-client/python/pull/782)
 
+**API Change:**
```

Comment:
why is this happening on the release branch, not on master?

Reply:
it was merged yesterday https://github.com/kubernetes-client/gen/pull/119 and gets included today since we pull the latest `kubernetes-client/gen`. It's a backwards-compatible fix IMO, so I keep it here. The master branch will sync up when we re-generate client there next time (11.0.0a1)

alternatively I can pin `kubernetes-client/gen` to exclude this change for this release. WDYT?

Reply:
ok. fine to include it.